### ISSUE TITLE
Embedded tweets not working on the blog

### ISSUE THREAD

**Issue by acabunoc** [mozilla/science.mozilla.org#531]
They show up on the wordpress instance: https://mozscienceblog.wpengine.com/?p=3657
![screen shot 2016-07-12 at 11 36 04](https://cloud.githubusercontent.com/assets/617994/16773167/d68155f8-4824-11e6-9fa0-8bcd6b01d377.png)

But not on the site:
https://science.mozilla.org/blog/why-developing-oss-matters

![screen shot 2016-07-12 at 11 36 27](https://cloud.githubusercontent.com/assets/617994/16773182/e4f65cf0-4824-11e6-8be6-0aa0d29dafa0.png)

**Comment by auremoser** [comment]
+1 to this, noted it for Anna's ISBE guest blog last week.

**Comment by alanmoo** [comment]
I think this might be due to the way WordPress exports HTML. There is a class we can use to recreate the styling, but it wouldn't be locked to Twitter's unless they offer a stylesheet I'm not aware of (which is totally possible).

**Comment by hannahkane** [comment]
As noted in roadmap doc, this is less important than other blog issues, given there's a workaround

**Comment by alanmoo** [comment]
By workaround, you mean some hardcoded CSS? I've got those in #555

**Comment by hannahkane** [comment]
Workaround is uploading a screenshot, I believe

**Comment by alanmoo** [comment]
That has some serious accessibility issues. @acabunoc How is [this](https://science-mozilla-org-sta-pr-555.herokuapp.com/blog/why-developing-oss-matters) as a workaround?

**Comment by acabunoc** [comment]
<3 much better! @alanmoo

**Comment by alanmoo** [comment]
Should we close this/change it now that #555's landed? The current implementation doesn't give complete tweet details (like conversation) but does get the tweet itself- [example](https://science.mozilla.org/blog/why-developing-oss-matters)

**Comment by acabunoc** [comment]
hey @alanmoo, the tweets in this post aren't formatted, are they embedded wrong? https://science.mozilla.org/blog/2016-isbe-review

**Comment by auremoser** [comment]
+1 to what abby said, I'm find with the single card view that @alanmoo linked to as an example, but we should document the correct embed format somewhere so it doesn't default to this:
![screen shot 2016-10-04 at 10 48 15 am](https://cloud.githubusercontent.com/assets/1559703/19078642/e0b4205a-8a1e-11e6-82a0-248e6954f365.png)

**Comment by alanmoo** [comment]
@acabunoc @auremoser I'm not sure how they were originally embedded (was the whole post copied and pasted from somewhere?) but if you paste a tweet link into the WordPress editor, it pulls in the tweet and will style it as in the previous example.

**Comment by auremoser** [comment]
Twitter has an "embed" generated iframe that it offers, I believe the unformatted tweets use that, because I think I assumed that you would need to embed a tweet and it wouldn't just generate formatting. Good to know, I'll update the post and test. :)

**Comment by alanmoo** [comment]
Yeah, this is kind of an intermediate step towards using that, but it was a quick and easy solution that looks good enough.
